nuxeo-drive-client/nxdrive/engine/engine.py

```diff
@@ -1005,3 +1009,29 @@ def fetch_all_users(self):
         except Exception as e:
             log.exception(e)
         return
+
+    def refresh_user(self, user_id):
```

Comment:
refresh_user method is called immediately after acquiring a doc_pair for syncing. If we do not have a _user_cache we have to query the database everytime acquire state is done. By using _user_cache we are limiting queries to sqlite database Users table to minimal. Sort of performance tuning that will help when syncing large number files (10000+ files).